src/Http/Validation/Rules/ShirtSize.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Http\Validation\Rules;
+
+use Respect\Validation\Rules\In;
+
+/**
+ * Validates whether the value is contained in the keys of the config item "tshirt_sizes".
+ */
+class ShirtSize extends In
+{
+    public function __construct()
```

Comment:
Having validation messages here (exists & wrong) might be helpful here as users might have none configured (change from disabled to enabled in config) or the wrong one (the organizer removed / changed some sizes after users have registered)?

Reply:
Field is required and there is an empty option for all shirt selects.